src/azurefluent/CodeGeneratorJvaf.cs

```diff
@@ -58,7 +58,10 @@ public class CodeGeneratorJvaf : CodeGeneratorJva
             {
                 // Manager
                 await WriteAzureServiceManagerJavaFile(codeModel).ConfigureAwait(false);
+            }
 
+            if (true == Settings.Instance.Host?.GetValue<bool?>("regenerate-pom").Result)

```

Comment:
Was there a reason it became more convenient to separate these out? They were one flag initially because generating a manager and POM were both part of scaffolding a new module. I'm not aware of cases where a project has one but not the other. At the same time it's also not a big deal to have to pass two flags--as long as I get on top of documenting them (hey, there's a good task to pick up).

Reply:
I wouldn't say it was more convenient, but rather it was more clear. There was no way to know that adding the flag `regenerate-pom` was the trigger to regenerate the managers as well. I'm fine with adding a flag called `generate-new-module` or something like that that triggers both `regenerate-pom` and `regenerate-managers`, but I think it should be more clear than it was.